pinot-common/src/test/java/org/apache/pinot/sql/parsers/CalciteSqlCompilerTest.java

```diff
@@ -2656,4 +2624,34 @@ public void testInvalidQueryWithAggregateFunction() {
     Assert.expectThrows(SqlCompilationException.class,
         () -> CalciteSqlParser.compileToPinotQuery("SELECT UPPER(col1), avg(col2) from foo"));
   }
+
+  /**
+   * Test for customized components in src/main/codegen/parserImpls.ftl file.
+   */
+  @Test
+  public void testParserExtensionImpl() {
+    SqlNode sqlNode = testSqlWithCustomSqlParser("INSERT INTO db.tbl "
+        + "FROM FILE 'file:///tmp/file1', FILE 'file:///tmp/file2'");
```

Comment:
maybe also add a test for INSERT with OPTIONS?

Reply:
I'm referring to something like:
```
INSERT INTO "baseballStats"
FROM FILE s3://<my-bucket>/baseballStats/rawdata/
OPTION(taskName=myTask-s3)
OPTION(input.fs.className=org.apache.pinot.plugin.filesystem.S3PinotFS)
OPTION(input.fs.prop.accessKey=<my-key>)
OPTION(input.fs.prop.secretKey=<my-secret>)
OPTION(input.fs.prop.region=us-west-2)
```
If it's already handled before then no problem from here.